source/dagger/gameplay/common/aiming_system.cpp

```diff
@@ -24,6 +24,20 @@ void AimingSystem::Run()
         {
             Float32 rotate = input_.values.at("rotate");        //Get the input value of the rotate command - the amount the angle changes if the button is pressed 
 
+        /*    
+        *            Example setup of for "rotate" command in input-context
```

Comment:
remove *, and move to the left a bit, like this:

Example setup of for "rotate" command in input-context:

Reply:
Also u can move this comment before this line:

 Float32 rotate = input_.values.at("rotate");

Reply:
Will do 👍